From: Jan Engelhardt <jengelh@inai.de>
Date: 2020-09-08 11:36:38.217829093 +0200

  * Drop %(date +%s) / __DATE__.
  * Drop forced stripping.
  * Replace lzma-sdk library name by what's used.
  * Since now the Makefile is already touched, enable use of system
    libraries through this patch rather than on the command-line.

---
 src/Makefile |   19 ++++++++++---------
 1 file changed, 10 insertions(+), 9 deletions(-)

Index: hashcat-6.1.1/src/Makefile
===================================================================
--- hashcat-6.1.1.orig/src/Makefile
+++ hashcat-6.1.1/src/Makefile
@@ -3,16 +3,16 @@
 ## License.....: MIT
 ##
 
-SHARED                  ?= 0
+SHARED                  ?= 1
 DEBUG                   := 0
 PRODUCTION              := 1
 PRODUCTION_VERSION      := v6.1.1
 ENABLE_CUBIN            ?= 1
 ENABLE_BRAIN            ?= 1
-USE_SYSTEM_LZMA         ?= 0
-USE_SYSTEM_ZLIB         ?= 0
+USE_SYSTEM_LZMA         ?= 1
+USE_SYSTEM_ZLIB         ?= 1
 USE_SYSTEM_OPENCL       ?= 0
-USE_SYSTEM_XXHASH       ?= 0
+USE_SYSTEM_XXHASH       ?= 1
 
 ##
 ## Detect Operating System
@@ -95,7 +95,7 @@ VERSION_PURE            := $(shell echo
 ##
 
 DESTDIR                 ?=
-PREFIX                  ?= /usr/local
+PREFIX                  ?= /usr
 
 INSTALL_FOLDER          ?= $(PREFIX)/bin
 SHARED_ROOT_FOLDER      ?= $(PREFIX)/share
@@ -112,7 +112,7 @@ LIBRARY_DEV_FOLDER      ?= $(LIBRARY_DEV
 ifeq ($(USE_SYSTEM_LZMA),0)
 DEPS_LZMA_PATH          := deps/LZMA-SDK/C
 else
-DEPS_LZMA_PATH          := $(LIBRARY_DEV_ROOT_FOLDER)
+DEPS_LZMA_PATH          := /usr/include/clzma
 endif
 
 ifeq ($(USE_SYSTEM_ZLIB),0)
@@ -158,7 +158,7 @@ endif # MSYS2
 ## Misc stuff
 ##
 
-COMPTIME                := $(shell date +%s)
+COMPTIME                := 0
 
 ##
 ## General compiler and linker options
@@ -166,6 +166,7 @@ COMPTIME                := $(shell date
 
 LFLAGS                  := $(LDFLAGS)
 
+CFLAGS += $(our_CFLAGS)
 ifeq ($(PRODUCTION),0)
 CFLAGS                  += -W
 CFLAGS                  += -Wall
@@ -188,7 +189,7 @@ endif
 ifeq ($(DEBUG),0)
 CFLAGS                  += -O2
 ifneq ($(UNAME),Darwin)
-LFLAGS                  += -s
+LFLAGS                  +=
 endif
 else
 ifeq ($(DEBUG),1)
@@ -214,7 +215,7 @@ CFLAGS                  += -pipe -std=gn
 # LZMA
 CFLAGS                  += -I$(DEPS_LZMA_PATH)
 ifeq ($(USE_SYSTEM_LZMA),1)
-LFLAGS                  += -llzmasdk
+LFLAGS                  += -lclzma
 endif
 
 # ZLIB
